### DISCUSSION TITLE
Is there a way to get the current working file's name and dir?

### DISCUSSION THREAD

**Discussion by linc01** [helix-editor/helix#6678] [General]
Hello,
Is there a way to get the current working file's name and dir? Same as the "file-name" carry in the config.toml file.
```
[editor.statusline]
left = ["mode", "spinner", "file-name"]
```

Everything is on the status line, but how can I read the "file-name" and use it in the terminal with `:append-output "termina_l command" "file-name"`

Thanks!

**Comment by gabydd** [comment]
That's: https://github.com/helix-editor/helix/pull/3393
